source/wp-content/plugins/wporg-developer-blog-cpts/wporg-developer-blog-cpts.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Plugin Name: Developer Blog custom post types and taxonomies.
+ * Plugin URI:  https://github.com/WordPress/wporg-developer-blog
+ * Description: Manages all of the custom post types and taxonomies for the dev blog.
+ * Version: 1.0.0
+ * Author: WordPress.org
+ * Author URI: https://wordpress.org/
+ * License:      GPL2
+ * License URI:  https://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain: wporg
+ */
+
+/**
+ * Register the CPT and tax for snippets.
```

Comment:
Minor change since this function has been updated:

```suggestion
 * Register the post types and taxonomies.
```